src/pages/base/markets/Markets.js

```diff
@@ -92,65 +110,80 @@ function Markets(props) {
                 align: "right",
               },
               {
-                dataField: "tvl_usd",
+                dataField: `tvl${fieldSuffix}`,
                 text: "TVL",
                 sort: true,
                 formatter: (cell, row) => (
                   <>
-                    <Value value={cell} decimals={2} prefix="$" compact />
+                    <Value
+                      value={cell}
+                      decimals={2}
+                      prefix={isTokenCurrency ? "" : "$"}
+                      compact
+                    />
                     <br />
                     <ValueChange
-                      value={cell - row["change"]["tvl_usd"]}
+                      value={cell - row["change"][`tvl${fieldSuffix}`]}
                       decimals={2}
-                      prefix="$"
+                      prefix={isTokenCurrency ? "" : "$"}
                       compact
                       icon
                       hideIfZero
-                      tooltipValue={row["change"]["tvl_usd"]}
+                      tooltipValue={row["change"][`tvl${fieldSuffix}`]}
                     />
                   </>
                 ),
                 headerAlign: "right",
                 align: "right",
               },
               {
-                dataField: "total_supply_usd",
+                dataField: `total_supply${fieldSuffix}`,
                 text: "Supply",
                 sort: true,
                 formatter: (cell, row) => (
                   <>
-                    <Value value={cell} decimals={2} prefix="$" compact />
+                    <Value
+                      value={cell}
+                      decimals={2}
+                      prefix={isTokenCurrency ? "" : "$"}
+                      compact
+                    />
                     <br />
                     <ValueChange
-                      value={cell - row["change"]["total_supply_usd"]}
+                      value={cell - row["change"][`total_supply${fieldSuffix}`]}
                       decimals={2}
-                      prefix="$"
+                      prefix={isTokenCurrency ? "" : "$"}
                       compact
                       icon
                       hideIfZero
-                      tooltipValue={row["change"]["total_supply_usd"]}
+                      tooltipValue={row["change"][`total_supply${fieldSuffix}`]}
                     />
                   </>
                 ),
                 headerAlign: "right",
                 align: "right",
               },
               {
-                dataField: "total_borrow_usd",
+                dataField: `total_borrow${fieldSuffix}`,
                 text: "Borrow",
                 sort: true,
                 formatter: (cell, row) => (
                   <>
-                    <Value value={cell} decimals={2} prefix="$" compact />
+                    <Value
+                      value={cell}
```

Comment:
`value={cell > 0 ? cell : "-"}` if it's 0.

Reply:
But then, we might as well do:
```
 if (
    value === undefined ||
    value === null ||
    (hideIfZero && value === 0) ||
    value === 0 ||
    value === 0.0
  ) {
    return "-";
  }
  
  in components/Value/Value.js and apply formatting to "-" everywhere where the value is 0, null, undefined, or 0.0 ?
```